### PR TITLE
Add git autoupdate config in labjs.

### PR DESCRIPTION
**Pull request by Piicksarn**

### CODE REVIEW

File: ajax/libs/labjs/package.json [hunk 1]
```diff
@@ -19,5 +19,14 @@
...
     }
-  ]
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/getify/LABjs.git",
+    "basePath": "",
+    "files": [
+      "LAB.js",
+      "LAB.min.js"
```
Comment (PeterDaveHello) [inline ajax/libs/labjs/package.json]: `LAB*.js` will be better.

**Comment by Piicksarn** [comment]
thanks, and i got it corrected !

**Comment by Piicksarn** [comment]
had it corrected and rebased to the master

**Comment by Piicksarn** [comment]
rebased it to the master

**Comment by Piicksarn** [comment]
[ Consider adding tags for the version 2.0.3 #105 ](https://github.com/getify/LABjs/issues/105)

**Pull request merged**